src/ItemOverlay/ItemOverlay.js

```diff
@@ -84,8 +84,10 @@ var ItemOverlay = module.exports = kind(
 	/**
 	* @private
 	*/
-	autoHideChanged: function() {
-		this.addRemoveClass('auto-hide', this.get('autoHide'));
+	autoHideChanged: function(was, is) {
+		if (this.autoHide) {
+			this.set('showing', (was || this.spotFocused));
```

Comment:
Yeah. I don't know what drugs I was on. :-) The amount of time that has passed between when I worked on this fix and when it was reviewed was long enough that I don't remember what I was thinking. Maybe I planned on setting spotFocused from the parent and forgot? Anyway, I reworked it and hopefully it's better now.